sema4ai/vscode-client/src/actionServer.ts

```diff
@@ -120,9 +121,21 @@ const startActionServerInternal = async (directory: Uri) => {
         return;
     }
 
+    const home = await getRobocorpHome();
     const env = createEnvWithRobocorpHome(await getRobocorpHome());
     env["RC_ADD_SHUTDOWN_API"] = "1";
 
+    const externalApiPid = path.join(home, "sema4ai-studio", "external-api-server.pid");
+    if (fs.existsSync(externalApiPid)) {
+        const fileContent = JSON.parse(fs.readFileSync(externalApiPid, "utf-8"));
+
+        if (fileContent.port) {
+            process.env["SEMA4AI_CREDENTIAL_API"] = `http://localhost:${fileContent.port}/api/v1/ace-services`;
```

Comment:
It should not set the `process.env`, rather it should just set the `env` in this function.